pkg/command/config.go

```diff
@@ -42,7 +41,9 @@ func init() {
 	)
 	serversCmd.AddCommand(listServersCmd)
 	addDeleteServersCmd()
-	command.DeprecateCommandWithAlternative(serversCmd, "1.2.0", "context")
+	// TODO: Update the plugin-runtime library with the new format and use the library method
+	msg := fmt.Sprintf("this was done in %q release, it will be removed following the deprecation policy (6 months). Use the %q command instead.\n", "0.90.0", "context")
```

Comment:
```suggestion
	msg := fmt.Sprintf("this was done in the %q release, it will be removed following the deprecation policy (6 months). Use the %q command instead.\n", "v0.90.0", "context")
```